.buildkite/bin/test

```diff
@@ -2,4 +2,4 @@
 
 set -eux
 
-docker-compose -f .buildkite/docker-compose.yml run widgets npm run test
+docker-compose -f .buildkite/docker-compose.yml run widgets npm run test-units
```

Comment:
Should we not delete this file (or have it run both test suites?)
(we had this conversation in the real world.... just documenting :)